### ISSUE TITLE
REQ: Include a diagram

### ISSUE THREAD

**Issue by dgomesbr** [bahrmichael/inbox-api#1]
ref: https://www.togaf-modeling.org/models/architecture-vision/solution-concept-diagrams.html

"A solution concept diagram provides a high-level orientation of the solution that is envisaged in order to meet the objectives of the architecture engagement. In contrast to the more formal and detailed architecture diagrams developed in the following phases, the solution concept represents a pencil sketch of the expected solution at the outset of the engagement. This diagram may embody key objectives, requirements and constraints for the engagement, and also highlight work areas to be investigated in more detail with formal architecture modeling. The purpose of this diagram is to quickly on-board and align stakeholders for a particular change initiative, so that all participants understand what the architecture engagement is seeking to achieve and how it is expected that a particular solution approach will meet the needs of the enterprise."

TL;DR: Diagrams are better when comes down to understanding the solution.

**Comment by bahrmichael** [comment]
Thank you, I addressed that with https://github.com/bahrmichael/inbox-api/commit/bfd0c2658c29f5bd83165f25eccf232d9e1f6fc1
